documentation/decisions/adr/2023-11-14-design-research-tools.md

```diff
@@ -0,0 +1,139 @@
+| Screeners    |  ✅      |  ❌           |  ✅         |  ✅             |  ✅            |  ❌        |  ❌       |  ✅         |  ✅            |
+| Intercepts   |  ✅      |  ❌           |  ✅         |  ✅             |  ✅            |  ❌        |  ❌       |  ❌         |  ❌            |
+| Data repo    |  🛠️      |  ❌           |  ✅         |  ✅             |  ✅            |  ✅        |  ❌       |  ❌         |  ❌            |
+| Nava exp.    |  ✅      |  ✅           |  ❌         |  ✅             |  ✅            |  ✅        |  ❌       |  ❌         |  ❌            |
+
+- ✅ Built-in
+- 🛠️ Integration available
+- ❌ Not included
+
+
+## Decision Outcome
+
+Chosen option: [Ethnio], because as a single tool it satisfies most decision drivers and has been validated by Nava experience.
+
+### Positive Consequences
+
+Ethnio is a user research CRM for tracking participant activity. Nava has used it with much success on other projects. It addresses many needs in one tool with the following features:
+
+- **Participant pool** — Manage participants with tags, notes, segments and filters. Keep a record of key interactions.
+- **Incentives** — Participants receive compensation in their preferred method. Weekly or monthly reports on spending. Prepaid balance never expires.
+- **Screeners** — Post or email screeners that recruit participants with qualifying questions and filtering logic and optional feedback surveys.
+- **Intercepts** — Embed targeted intercepts in desktop and mobile viewports to recruit or survey site users in specific locations or devices.
+- **Scheduling** — Invite, schedule, and remind participants. Combined team calendar with Outlook and Google integration.
+
+There is potential (depending on procurement) for Ethnio to be used both by this Simpler.Grants.gov project and HHS's other grants improvement efforts, which would let them share a single participant pool within a centralized database where participants can be tagged/labeled and their activity can be tracked (incentives might also be funded in a single location).
+
+Ethnio has the ability to create surveys of various types (open-ended feedback, targeted, screening potential participants), which could also satisfy the ["Method & technology for collecting targeted feedback" ADR issue](https://github.com/HHS/simpler-grants-gov/issues/306).
+
+Other features of note:
+- Internationalized and localized — [Ethnio is fully i18n-friendly](https://help.ethn.io/article/118-international-ethnio-using-other-languages-in-screeners-scheduling-and-incentives). Screeners, intercepts, and inentives/scheduling emails can all be given manual translations through advanced options in the editor (with the exception of right-to-left support).
+- Section 508 and WCAG 2.0 (AA)
+- GDPR- and CCPA-compliant data retention, expiration, and portability
+
+**Cost:** $82+/seat/month, an Enterprise account is required for Participant Pool feature
```

Comment:
How many seats do we expect we would need in the platform?

Reply:
```suggestion
**Cost:** $82+/seat/month (note: The Enterprise account is required for Participant Pool feature. It has an 8 seat minimum, which should more than suffice)
```